aws_vars.yml

```diff
@@ -26,40 +26,32 @@ aws_access_key: "{{ec2_access_key}}"
 aws_secret_key: "{{ec2_secret_key}}"
 
 ## Set the AMI IDs here (or override with -e). Some commonly used AMIs are defined here, but ultimately the ones that matter are 'tower_ami_id' and 'ocp_ami_id'
-## Tower AMI unconfigured
-tower_ami_id_unconfigured: ami-29610c4a
-## Tower AMI ID fully configured
-tower_ami_id_configured: ami-800a29e5
 ## Tower AMI ID - This is the only variable used in the Playbooks, set it to one of the above (or override with -e)
-tower_ami_id: "{{ tower_ami_id_unconfigured }}"
-## OCP 3.5 ID
-ocp_ami_id_3_5: ami-3934175c
-## OCP 3.6 ID
-ocp_ami_id_3_6: ami-7975151a
+tower_ami_id: "{{ tower_ami_id }}"
 ## OCP ID - This is the only variable used in the Playbooks, set it to one of the above (or override with -e)
-ocp_ami_id: "{{ ocp_ami_id_3_6 }}"
+ocp_ami_id: "{{ ocp_ami_id }}"
 
 ## AWS Info
 tower_inst_type: t2.medium
 ocp_master_inst_type: t2.large
 ocp_node_inst_type: t2.xlarge
 # The subnet_id can be dynamically set if initially creating it at the same time with the aws_vpc_keypair.yml playbook. Setting is statically for now
-aws_subnet_id: subnet-dba04d92
-aws_region: ap-southeast-1
+aws_subnet_id: "{{ aws_subnet_id }}"
```

Comment:
Absolutely love this approach, there is no reason for us to hard code specific environment details in here. We should just share the details of various region info in separate secrets files and share those out of band (gdoc or other way). However, I would probably populate these variables in the `my_secrets.yml` to show people what they should be defining.

Reply:
+1

Reply:
added an example my_secrets.yml file